src/core/ir/io.rs

```diff
@@ -167,12 +166,18 @@ impl IO {
 }
 
 impl<'a, Ctx: ResolverContextLike<'a> + Sync + Send> CacheKey<EvaluationContext<'a, Ctx>> for IO {
-    fn cache_key(&self, ctx: &EvaluationContext<'a, Ctx>) -> Option<IoId> {
+    fn cache_key(&self, ctx: &EvaluationContext<'a, Ctx>) -> IoId {
         match self {
             IO::Http { req_template, .. } => req_template.cache_key(ctx),
             IO::Grpc { req_template, .. } => req_template.cache_key(ctx),
             IO::GraphQL { req_template, .. } => req_template.cache_key(ctx),
-            IO::Js { .. } => None,
+            IO::Js { name } => {
+                let mut hasher = TailcallHasher::default();
+                let val = ctx.value().map(|v| v.to_string()).unwrap_or_default();
```

Comment:
This hashing technique is going to be very very slow. We are effectively serializing the data twice.

Reply:
but const value don't have Hash implemented